packages/terra-profile-image/tests/jest/ProfileImage.test.jsx

```diff
@@ -6,3 +6,9 @@ it('should render a Terra image with a placeholder prop', () => {
   const wrapper = shallow(<ProfileImage src="profile.jpg" height="75" width="75" alt="A profile image" />);
   expect(wrapper).toMatchSnapshot();
 });
+
+/* Profile Image source is not provided */
+it('should render a Terra image without Source prop', () => {
```

Comment:
`terra-image` is a component that comes into picture in a success scenario. I'd not mention it here in the test description for a scenario when `src` is not provided.